apps/client/src/pages/CyclesHistory.jsx

```diff
@@ -0,0 +1,181 @@
+      "Spore Lot": c.spore?.lot || "",
+      "Spore Exp": c.spore?.expireDate
+        ? formatDateTime(c.spore.expireDate)
+        : "",
+      "Spore Incubated": c.spore?.incubatedAt
+        ? formatDateTime(c.spore.incubatedAt)
+        : "",
+      "Spore Result": c.spore?.result || "",
+      "Spore Verified By": c.spore?.verifiedBy || "",
+    }));
+
+    const csv = toCSV(data);
+    const name = `cycles-${machine?.name || id}.csv`;
+    downloadFile(csv, name, "text/csv");
+  }
+
+  return (
+    <div>
+      {/* Header Bar */}
+      <div style={headerBar}>
+        <h1 style={{ margin: 0 }}>Cycles — {machine ? machine.name : "…"}</h1>
+        <div style={{ display: "flex", gap: 8 }}>
+          <button onClick={exportCSV} style={btnGhost}>
+            Export CSV
+          </button>
+          <Link to={`/machines/${id}`} style={linkBtn}>
+            Back to machine
+          </Link>
+          <Link to={`/cycles?machineId=${id}`} style={linkBtn}>
+            Log cycle
+          </Link>
+        </div>
+      </div>
+
+      {err && (
+        <div style={{ color: "var(--color-danger)", marginBottom: 12 }}>
+          Failed to load: {err}
+        </div>
+      )}
+
+      {/* Table */}
+      <div style={tableWrap}>
+        <table style={table}>
+          <thead style={thead}>
+            <tr>
+              <th style={th}>Load #</th>
+              <th style={th}>Started</th>
+              <th style={th}>Completed</th>
+              <th style={th}>Result</th>
+              <th style={th}>Items</th>
+            </tr>
+          </thead>
+          <tbody>
+            {loading ? (
+              <tr>
+                <td colSpan={5} style={{ ...td, opacity: 0.7 }}>
+                  Loading…
+                </td>
+              </tr>
+            ) : rows.length ? (
+              rows.map((r) => (
+                <tr key={r._id} style={tr}>
+                  <td style={td}>{r.loadNumber || "—"}</td>
+                  <td style={td}>{formatDateTime(r.startedAt)}</td>
+                  <td style={td}>{formatDateTime(r.completedAt)}</td>
+                  <td style={td}>{r.result || "—"}</td>
+                  <td style={{ ...td, color: "var(--color-text-muted)" }}>
+                    {r.items || "—"}
+                  </td>
+                </tr>
+              ))
+            ) : (
+              <tr>
+                <td colSpan={5} style={{ ...td, opacity: 0.7 }}>
+                  No cycles yet.
+                </td>
+              </tr>
+            )}
+          </tbody>
+        </table>
+      </div>
+    </div>
+  );
+}
+
+/* ---- styles (matches MaintenanceHistory look) ---- */
+const headerBar = {
+  display: "flex",
+  justifyContent: "space-between",
+  alignItems: "center",
+  marginBottom: 16,
+};
+
+const linkBtn = {
+  color: "#cbd5e1",
+  textDecoration: "none",
+  border: "1px solid var(--color-border)",
+  borderRadius: 12,
+  padding: "8px 12px",
+};
+
+const btnGhost = {
```

Comment:
According to the criteria, you need to use `BEM` classes. You should store them in `.css` files